data/json/items/gun/762.json

```diff
@@ -30,6 +30,38 @@
     "flags": ["MODE_BURST", "NEVER_JAMS"]
   },
   {
+    "id": "arx160",
+    "type": "GUN",
+    "symbol": "(",
```

Comment:
Yes, I have been using such a `jq` script and then running the results through a formatter and then manually editing to improve the readability. The `jq` script also drops defaulted values. Perhaps I should publish this?